### PR TITLE
Pj/feat/seed cafes function

### PR DESCRIPTION
**Pull request by pablisch**
This function is basically completed but cannot be manually tested until the cafeModel is completed and seed data is in place.

- [x] Connect to conditional database.
- [x] Delete current data.
- [x] Insert new data.
- [x] log result.
- [x] Added scripts for seeding in package.json.
- [x] Updated README with seeding instructions.

Still to do at this time:
- [x] cafeModel complete
- [x] seed data file complete
- [x] seeding function tested

### CODE REVIEW

**Comment by kikilondon** [comment]
I have updated the model on MongoDB with your feedback. I will wait for @siqbal181 today at 5 pm to discuss about it. If you want to join us, we will be in room 1, you are welcome.

**Comment by kikilondon** [comment]
🙌

File: README.md [hunk 1]
```diff
@@ -63,6 +63,15 @@ At time of writing, the databases contain only a health collection as referenced
...
 
+## Seeding the database collections
+
+The database collections can be seeded with using the `seedCafes.js` file in the `backend` directory.
+
+`cd backend` to navigate to the backend directory.
+
+`npm run seed:cafes:dev` will seed the `cafes` collection in the `aura_DEV` database.
+`npm run seed:cafes:test` will seed the `cafes` collection in the `aura_TEST` database.
+
```
Comment (kikilondon) [inline README.md:74]: Add documentation for seeding the production database

**Review by kikilondon — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (pablisch) [reply README.md:74]: Requested changes have been made.

**Comment by pablisch** [comment]
Requested changes have been made.

**Review by kikilondon — APPROVED** [review]

**Pull request merged**